executor.py

```diff
@@ -147,4 +140,22 @@ def fill_embedding(self, docs: Optional[DocumentArray], **kwargs):
             return
 
         for doc in docs:
-            doc.embedding = self._storage[doc.id].embedding
+            doc.embedding = self._index[doc.id].embedding
+
+    @requests(on='/dump')
+    def dump(self, docs: Optional[DocumentArray]=None, **kwargs):
+        """dump indexed Documents to disk
+        """
+        if docs:
+            self._index.extend(docs)
+        bytes = self._index.to_bytes(protocol=self.protocol, compress=self.compress)
+        with open(os.path.join(self.workspace, SimpleIndexer.FILE_NAME), 'wb') as f:
+            f.write(bytes)
```

Comment:
What is the purpose of `/dump`? To my understanding, we want to dump the indexed Documents to the disk. If this is the case, we don't need `docs` as inputs.

Reply:
Correct; we misremembered the required parameters for a request-method. Will change